api/src/controllers/resumeReview/patchAllResumeReview.ts

```diff
@@ -52,6 +53,14 @@ const patchAllResumeReview = controller(async (req: Request<Params, unknown, Req
     await new ParamsValidator().validateAndThrow(req.params);
     await new ReqBodyValidator().validateAndThrow(req.body);
 
+    // Ensure that you can't claim a resume review claimed by someone else
+    if (req.body.reviewer !== undefined) {
+        const rr = await resumeReviewRepository.get(req.params.resumeReview);
+        if (req.body.reviewer !== null && rr[0].reviewer_id !== null) {
+            throw new BadRequestException({ issue: 'You must set the reviewer to null before you can change it to another user.' });
+        }
+    }
```

Comment:
Nit: I prefer this but this works just fine, because of how we're using the API, `PATCH` with a reviewer field always implies claiming so all good 👍

```suggestion
    if (req.body.state === 'reviewing') {
        const rr = await resumeReviewRepository.get(req.params.resumeReview);
        if (rr[0].reviewer_id !== req.body.reviewer) {
            throw new BadRequestException({ issue: 'You must set the reviewer to null before you can change it to another user.' });
        }
    }
```

Reply:
Good point, since this is functionally the same I'm just going to leave it for now though.

Reply:
Agreed, I'll fix the tests